src/backend/dx12/src/lib.rs

```diff
@@ -320,14 +320,30 @@ impl hal::PhysicalDevice<Backend> for PhysicalDevice {
 
     fn format_properties(&self, fmt: Option<f::Format>) -> f::Properties {
         let idx = fmt.map(|fmt| fmt as usize).unwrap_or(0);
-        format::query_properties()[idx]
+        self.format_properties[idx]
     }
 
     fn image_format_properties(
         &self, format: f::Format, dimensions: u8, tiling: image::Tiling,
         usage: image::Usage, storage_flags: image::StorageFlags,
     ) -> Option<image::FormatProperties> {
         conv::map_format(format)?; //filter out unknown formats
+        let format_desc = format.surface_desc();
+        // transfers are tricky, here is a list of exclusions
+        if usage.intersects(image::Usage::TRANSFER_SRC | image::Usage::TRANSFER_DST) {
```

Comment:
Couldn't we just check here if the requested flags are supported properties of the format?